src/forms/login-popup/index.jsx

```diff
@@ -93,6 +94,9 @@ const LoginForm = () => {
     } else if (thirdPartyAuthApiStatus === FAILURE_STATE && accountActivation === null) {
       emailOrUsernameRef.current.focus();
     }
+    if (moveScrollToTop) {
+      moveScrollToTop(loginFormHeadingRef, 'end');
```

Comment:
Suppose the user lands on this screen and scrolls down the screen. Assume the MFE context API call is in a pending state and when it's done the screen will be scrolled up due to `thirdPartyAuthApiStatus` is added in useEffect.

Reply:
done

Reply:
We need to remove `moveScrollToTop` function from this `useEffect`, otherwise issue exists.